_observing-your-data/trace/ta-dashboards.md

```diff
@@ -1,25 +1,143 @@
-To drill down on the traces that make up a trace group, choose the number of traces in the column on the right. Then choose an individual trace for a detailed summary.
+## Setting up the OpenTelemetry Demo
 
-![Detailed trace view]({{site.url}}{{site.baseurl}}/images/ta-trace.png)
+Set up the OpenTelemetry Demo with OpenSearch, which simulates a distributed application generating real-time telemetry data. This demo provides a practical environment to explore the Trace Analytics plugin's features before implementation.
 
-The **Services** view lists all services in the application, plus an interactive map that shows how the various services connect to each other. In contrast to the dashboard, which helps identify problems by operation, the service map helps identify problems by service. Try sorting by error rate or latency to get a sense of potential problem areas of your application.
+**Step 1: Set up OpenTelemetry Demo**
+  
+  - Clone the [OpenTelemetry Demo with OpenSearch](https://github.com/opensearch-project/opentelemetry-demo) repository: `git clone https://github.com/opensearch-project/opentelemetry-demo`.
+  - Follow the [Getting Started](https://github.com/opensearch-project/opentelemetry-demo/blob/main/tutorial/GettingStarted.md) instructions to deploy the demo application using Docker, which runs multiple microservices generating telemetry data.
 
-![Service view]({{site.url}}{{site.baseurl}}/images/ta-services.png)
+**Step 2: Ingest telemetry data**
+
+  - Configure the OTel collectors to send telemetry data (traces, metrics, logs) to your OpenSearch cluster, using the [pre-existing setup](https://github.com/opensearch-project/opentelemetry-demo/tree/main/src/otelcollector).
+  - Confirm that [Data Prepper](https://github.com/opensearch-project/opentelemetry-demo/tree/main/src/dataprepper) is set up to process the incoming data, handle trace analytics and service map pipelines, submit data to required indexes, and perform pre-aggregated calculations.
+
+**Step 3: Explore Trace Analytics in OpenSearch Dashboards**
+
+The **Trace Analytics** application includes two options: **Services** and **Traces**:
+
+  - **Services** lists all services in the application, plus an interactive map that shows how the various services connect to each other. In contrast to the dashboard (which helps identify problems by operation), the **Service map** helps you identify problems by service based on error rates and latency. To access this option, go to **Trace Analytics** > **Services**.
+  - **Traces** groups traces together by HTTP method and path so that you can see the average latency, error rate, and trends associated with a particular operation. For a more focused view, try filtering by trace group name. To acces this option, go to **Trace Analytics** > **Traces**. From the **Trace Groups** panel you can drill down on the traces that make up a trace group. From the **Traces** panel you can drill down on individual traces for a detailed summary.
```

Comment:
```suggestion
  - **Traces** groups traces together by HTTP method and path so that you can see the average latency, error rate, and trends associated with a particular operation. For a more focused view, try filtering by trace group name. To access this option, go to **Trace Analytics** > **Traces**. From the **Trace Groups** panel, you can drill down on the traces that comprise a trace group. From the **Traces** panel you can drill down on individual traces for a detailed summary.
```